### PR TITLE
Add mpc package

### PR DESCRIPTION
**Pull request by mklein994**
mpc is a command line client for the Music Player Daemon (MPD).

references termux/termux-packages/#536

### CODE REVIEW

File: packages/mpc/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pkgname=mpc
+TERMUX_PKG_HOMEPAGE=https://www.musicpd.org/clients/mpc/
+TERMUX_PKG_DESCRIPTION="Minimalist command line interface for MPD"
+TERMUX_PKG_DEPENDS="libmpdclient"
+_MAIN_VERSION=0
+_PATCH_VERSION=28
+TERMUX_PKG_VERSION=${_MAIN_VERSION}.${_PATCH_VERSION}
+TERMUX_PKG_SRCURL=https://www.musicpd.org/download/${pkgname}/${_MAIN_VERSION}/${pkgname}-${_MAIN_VERSION}.tar.xz
```
Comment (fornwall) [inline packages/mpc/build.sh]: I guess this should use the patch version, as in:

    TERMUX_PKG_SRCURL=https://www.musicpd.org/download/${pkgname}/${_MAIN_VERSION}/${pkgname}-${_MAIN_VERSION}.${_PATCH_VERSION}.tar.xz

**Comment by fornwall** [comment]
Could you add

    TERMUX_PKG_MAINTAINER="Matthew Klein @mklein994"

if you want to maintain the package (will be used for future reference, so that issues in the package can be assigned to you)?

**Comment by mklein994** [comment]
Done for both libmpdclient  (#537) and this one. Looking back, I probably should have just combined both pull requests into one, just to simplify things a bit.

**Comment by fornwall** [comment]
Thanks a lot! Merged and available in the package repository.

**Pull request merged**